wormhole-connect/src/config/types.ts

```diff
@@ -28,7 +28,30 @@ import RouteOperator from 'routes/operator';
 import { UiConfig } from './ui';
 import { TransferInfo } from 'utils/sdkv2';
 
-export enum Icon {
+export enum ChainIcon {
```

Comment:
can we use the `Chain` type from the SDK instead of creating this new thing?

Reply:
But that's a type and TBH I prefer enum when mapping the icon for the chain in `icons/ChainIcons.tsx`. If you like we can track this further in a separate ticket?

Reply:
dont worry about TokenIcon. I'm obliterating that code anyway in my arbitrary token branch.

Reply:
there is already an enum for chains, it's `Chain` in the SDK. if we can use that we should

Reply:
Do you mean `import { Chain } from '@wormhole-foundation/sdk';`? It seems to be a string type. We can still use that with a string => icon map:
```
const iconMap: { [Chain]: React.JSX.Element } = {
  'GLMR': GLMR(),
  'AVAX': AVAX(),
  'BSC': BSC(),
  ...
};

<ChainIcon icon={chainConfig?.symbol} />
```
Or did you mean `ChainId` as the enum for chains? I wanted to double-check if what you mean is something else.

Reply:
and there is `chains` from `@wormhole-foundation/sdk` as well, which seems more like it.